egs/wsj/s5/utils/data/perturb_data_dir_speed_3way.sh

```diff
@@ -38,9 +38,17 @@ utils/data/get_utt2dur.sh ${srcdir}
 
 utils/data/perturb_data_dir_speed.sh 0.9 ${srcdir} ${destdir}_speed0.9 || exit 1
 utils/data/perturb_data_dir_speed.sh 1.1 ${srcdir} ${destdir}_speed1.1 || exit 1
-utils/data/combine_data.sh $destdir ${srcdir} ${destdir}_speed0.9 ${destdir}_speed1.1 || exit 1
 
-rm -r ${destdir}_speed0.9 ${destdir}_speed1.1
+utils/copy_data_dir.sh --spk-prefix sp1.0- --utt-prefix sp1.0- ${srcdir} data/temp0
```

Comment:
since you are already creating the dir with _speed0.9 and  _speed1.0 as suffixes, I think it would be clearer and more consistent to use _speed1.0 as the suffix for this.